### PR TITLE
add note about licensing

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/940

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/usage-analytics/enable-usage-analytics.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ If you don't already have Power BI, you can [sign up for the free Power BI servi
 ![Select Get started free to get Power Bi](../media/7916b7ac-c1c8-40c0-b076-9a96ef5e0eb7.png)
   
 You can also expand **Products** to buy a version of Power BI. 
+
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline Office365-Admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
> [!NOTE]
```

File: Office365-Admin/usage-analytics/enable-usage-analytics.md [hunk 2]
```diff
@@ -36,6 +36,11 @@ If you don't already have Power BI, you can [sign up for the free Power BI servi
...
+>You need a Power BI Pro license to install, customize, and distribute a template app. Please see more at [Prerequisites](https://docs.microsoft.com/en-us/power-bi/service-template-apps-install-distribute?source=docs#prerequisites)
```
Comment (JohanFreelancer9) [inline Office365-Admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
> You need a Power BI Pro license to install, customize, and distribute a template app. For more information, please see [Prerequisites](https://docs.microsoft.com/power-bi/service-template-apps-install-distribute?source=docs#prerequisites).
```

File: Office365-Admin/usage-analytics/enable-usage-analytics.md [hunk 3]
```diff
@@ -36,6 +36,11 @@ If you don't already have Power BI, you can [sign up for the free Power BI servi
...
+
+>You need a Power BI Pro license to share your content, and the people you share it with do too, or the content needs to be in a workspace in a [Premium capacity](https://docs.microsoft.com/en-us/power-bi/service-premium-what-is).
```
Comment (JohanFreelancer9) [inline Office365-Admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
> You need a Power BI Pro license to share your content, and the people you share it with do too, or the content needs to be in a workspace in a [Premium capacity](https://docs.microsoft.com/power-bi/service-premium-what-is). 
```

File: Office365-Admin/usage-analytics/enable-usage-analytics.md [hunk 4]
```diff
@@ -85,4 +90,4 @@ To make the data that is collected for all reports anonymous, you have to be a g
     
 2. Select **Reports**, and then choose to **Display anonymous identifiers**. This setting gets applied both to the usage reports as well as to the template app.
   
-3. Select **Save changes**.S
\ No newline at end of file
+3. Select **Save changes**.S
```
Comment (JohanFreelancer9) [inline Office365-Admin/usage-analytics/enable-usage-analytics.md]:
```suggestion
3. Select **Save changes**.
```

**Comment by mypil** [comment]
@Ajayan1008 @Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #940

Thank you.

cc: @kenwith

**Pull request merged**